utility/utils.py

```diff
@@ -1884,12 +1884,34 @@ def clone_the_repo(config, node, path_to_clone):
 
 
 def calculate_available_storage(node):
-    log.info("Calculate current storage present in cluster")
-    out, err = node.exec_command(cmd="sudo ceph df --format json")
     import json
 
+    log.info(
+        "Calculate maximum storage that is available to be used."
+        + " It is the amount of data that can be used before the first OSD becomes full."
+        + " This is implicitly divided by replication factor or erasure code"
+    )
+    out, err = node.exec_command(cmd="sudo radosgw-admin zone get --format json")
     out = json.loads(out)
-    return out["stats"]["total_avail_bytes"]
+    zone_name = out["name"]
+    rgw_bucket_data_pool = f"{zone_name}.rgw.buckets.data"
+    out, err = node.exec_command(cmd="sudo ceph df --format json")
+    if rgw_bucket_data_pool not in out:
+        node.exec_command(cmd=f"sudo ceph osd pool create {rgw_bucket_data_pool} 64 64")
```

Comment:
is there a reason to create the data pool with 64 pgs?

Reply:
@viduship , based on our discussion, removed the pgnum value, allowing it to use default as 8 since autoscaler is on